algorithms/Python/strings/roman-to-int.py

```diff
@@ -0,0 +1,41 @@
+"""
+This program converts a string that represents a valid roman number to its decimal equivalent.
+If the string is not a valid roman number, raises a ValueError exception.
+Time Complexity : O(n)
+Space Complexity : O(1)
+"""
+
+equivalence = {'I': 1, 'V': 5, 'X': 10, 'L': 50, 'C': 100, 'D': 500, 'M': 1000}
+
+def is_valid_roman_string(roman_number: str) -> bool:
+    """Returns True if all characters of the given string are valid roman numbers"""
+    return set(roman_number).issubset(equivalence.keys())
+
+
+def convertion(roman_number: str) -> int:
```

Comment:
```suggestion
def conversion(roman_number: str) -> int:
```
./algorithms/Python/strings/roman-to-int.py:15: convertion ==> conversion